ably.d.ts

```diff
@@ -651,8 +651,7 @@ declare namespace Types {
 		isLast: () => boolean;
 	}
 
-	class HttpPaginatedResponse extends PaginatedResult<any> {
-		items: string[];
+	class HttpPaginatedResponse<T = any> extends PaginatedResult<T> {
 		statusCode: number;
```

Comment:
You accidentally deleted the `items` line instead of changing it to `items: T[];`..?

Reply:
`PaginatedResult` actually has it's own `items` property which is already typed as `T[]` so deleting this line has the same effect but without duplication

Reply:
ah 👍